### PR TITLE
added FAQ how to contribute

### PR DESCRIPTION
**Pull request by lefherz**
addresses #350

I'm not totally sure whether we need this. Our channels are already addressed at contact.rst. Still not bad to have it fledged out, if there is no specific reason not to include it, let's put it in.

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -131,6 +131,20 @@ For encrypted mailing lists like `schleuder
 Help welcome.
 
 
+How can I participate in Autocrypt?
+-----------------------------------
+
+First of all it would be nice if you join the discussion on IRC.
+Autocrypt lives in #autocrypt on irc.freenode.net, if you have ideas,
+discuss them there first.
```
Comment (hpk42) [inline doc/faq.rst]: i wouldn't make a hierarchy here but simply say "To ask questions or make suggestions, you can join the ML or join the more informal chat in the #autocrypt room on irc.freenode.net".

File: doc/faq.rst [hunk 2]
```diff
@@ -131,6 +131,20 @@ For encrypted mailing lists like `schleuder
...
+
+Apart from that, anyone can open a pull request to the spec or to the
+website. Feel free to share your thoughts and skills!
```
Comment (hpk42) [inline doc/faq.rst]: I'd continue with using "you" here like in the first paragraph, so something like "We appreciate if you open a pull request to the spec or to the website".

File: doc/faq.rst [hunk 3]
```diff
@@ -131,6 +131,20 @@ For encrypted mailing lists like `schleuder
...
+
+There are sub-teams in autocrypt, such as the press team, the CoC
+team, or the GitHub organization. To become a part and contribute
+to them, just ask on IRC. If people trust you, you will be admitted.
```
Comment (hpk42) [inline doc/faq.rst]: Is it realistic that someone drops by, has not done anything yet and directly becomes part of press or coc-team?  I wouldn't talk directly about admission but about getting started and rather say "after some time, you can also ask for becoming a more formal part of any of the teams".

**Comment by lefherz** [comment]
This PR contains barely more than contact information, and that is redundant, as we have a contact page. closed.

**Pull request closed without merging**